server/routes/shared_api/place.py

```diff
@@ -233,22 +233,44 @@ def get_named_typed_place():
   return Response(json.dumps(ret), 200, mimetype='application/json')
 
 
-@bp.route('/stat-vars/union', methods=['POST'])
-def get_stat_vars_union():
+@bp.route('/variable', methods=['GET', 'POST'])
+def get_place_variable():
   """Get all the statistical variables that exist for some places.
 
   Returns:
       List of unique statistical variable dcids each as a string.
   """
+  dcids = request.args.getlist('dcids')
+  if not dcids:
+    dcids = request.json['dcids']
+  resp = fetch.entity_variables(dcids)
   result = []
-  entities = sorted(request.json.get('dcids', []))
-  resp = dc.entity_variables(entities)
-  for var, entity_obs in resp['byVariable'].items():
-    if len(entity_obs.get('byEntity'), {}) == len(entities):
+  for var, entity_obs in resp.items():
+    if len(entity_obs) == len(dcids):
```

Comment:
sorry I didn't catch this in the earlier PR, but for stat var union, the stat var just has to exist for at least one of the dcids: https://github.com/datacommonsorg/mixer/blob/master/internal/server/statvar/place_stat_vars.go#L146

Reply:
Updated.